filebeat/_meta/common.reference.p2.yml

```diff
@@ -270,6 +270,11 @@ filebeat.inputs:
 # Default is 0, not waiting.
 #filebeat.shutdown_timeout: 0
 
+# Defines if Filebeat pipelines have to be updated on Ingest Node.
```

Comment:
As this is not expected to be used in production I would prefer to not have it in the config file. 

What about `filebeat pipelines update` as a command? I'm hoping like this the config option would also not have to be passed through all levels.

Reply:
My problem with your proposal is the user would need run two commands, instead of one.
What if it would be a command line flag? `--update-pipelines` or something like that. So it wouldn't be in the config file and users wouldn't need to run two commands.

Reply:
I like the idea of having it as a command line flag. Normally I'm oposed to have command line flags which are not in the docs but I think this one is a good example where it makes a sense.

It also should allow you to make the code more local.

Reply:
I assume that change is not needed anymore?